docker/Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN mkdir -p ${LOG_PATH} \
 # apache2-utils is required to execute htpasswd
 RUN apt-get update \
     && apt-get install -y --no-install-recommends apt-utils libsasl2-dev libssl-dev netcat-traditional \
-    vim libldap2-dev libfuzzy-dev net-tools python3-psycopg2 git apache2-utils \
+    vim libldap2-dev libfuzzy-dev net-tools python3-psycopg2 git apache2-utils tshark \
```

Comment:
please comment that tshark is required for this analyzer otherwise we will forget it